### DISCUSSION TITLE
PWAsForFirefox 2.4.1

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#290] [Announcements]
## Changed

* **Enabled unified extensions (#288):**
  Unified extensions (introduced in Firefox 109) are now enabled by default in the app browser. This means extension widgets cannot be removed or moved into the overflow menu anymore. Instead, all unpinned extensions can be accessed from the unified extensions button. This may change in the future.
  In comparison to normal Firefox, the unified extension button can also be (re)moved. It is, however, recommended not to remove it, as this prevents accessing non-pinned extensions and may cause problems.

## Fixed

* Some inconsistencies with settings.
* Layout alignment in the app browser toolbar (#281).
* Parsing manifests that include media ranges as types (#275).

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v2.4.1'>PWAsForFirefox 2.4.1</a>.</em>

**Comment by xmha97** [comment]
Thanks
